src/clang.rs

```diff
@@ -638,35 +659,38 @@ impl Cursor {
         }
     }
 
-    /// Whether this cursor has the `warn_unused_result` attribute.
-    pub fn has_warn_unused_result_attr(&self) -> bool {
-        // FIXME(emilio): clang-sys doesn't expose this (from clang 9).
-        const CXCursor_WarnUnusedResultAttr: CXCursorKind = 440;
-        self.has_attr("warn_unused_result", Some(CXCursor_WarnUnusedResultAttr))
-    }
+    pub fn has_attrs<const N: usize>(
```

Comment:
@emilio I think maybe checking for several attributes in a single visit pass might be better, let me know what you think.